Editor/Scripts/customize_mesh_asset_processing.py

```diff
@@ -0,0 +1,62 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+import sys
+import azlmbr
+
+from PySide2.QtWidgets import QMenuBar
+
+from tutorial import Tutorial, TutorialStep
+
+class CustomizeMeshAssetProcessingTutorial(Tutorial):
+    def __init__(self):
+        super(CustomizeMeshAssetProcessingTutorial, self).__init__()
+
+        self.title = "Customize Mesh Asset Processing"
+
+        self.add_step(TutorialStep("Customize Mesh Asset Processing", 
+                """<html><p style="font-size:13px">Greetings!<br><br><i> Meshes </i> refer to the external appearances
+                of objects.<br>In this tutorial, we'll edit the mesh of a sphere by enlarging it. We'll also introduce
+                some other ways you can edit the mesh. <br><br>Click next to continue.</p></html>"""))
+        self.add_step(TutorialStep("Preparing the scene", """<html><p style="font-size:13px">The <b>Asset Processor </b> runs 
+                in the background automatically detecting source assets and scheduling process jobs for them.
+                <br><br>In <b>Asset Processor</b>, search for the <b>sphere.fbx</b> source asset and 
```

Comment:
This should say `Asset Browser` instead of `Asset Processor`. The `Asset Processor` reference in the first sentence of this paragraph is still valid.